src/js/components/NestedServiceLinks.js

```diff
@@ -46,11 +46,8 @@ class NestedServiceLinks extends React.Component {
       minorLinkAnchorClassName
     );
     let nestedGroups = this.getServicePathParts();
-    let popCount = 1;
+    const popCount = 1;
```

Comment:
Could just inline this because only used once, and this variable name is confusing.